pom.xml

```diff
@@ -172,6 +172,11 @@ questions.
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.8.0</version>
+                    <configuration>
+                        <compilerVersion>1.8</compilerVersion>
+                        <source>1.8</source>
+                        <target>1.8</target>
```

Comment:
Don't know maven much, is this going to be inherited to `<build>` in child poms?

Reply:
Yes, inheritance in submodules is one of the points of `<pluginManagement>`.